### PR TITLE
Add list for About this app screen

### LINKED ISSUE #260: Add list for About this app screen
## Kind (Required)

- Improvement

 ## Overview (Required)

 - Please add list to About screen.
 - No content is needed.
 - Please add to here
image
image

 ## Links	

-

### PR DESCRIPTION
**Pull request by akihiro-n**
## Issue
- close #260 

## Overview (Required)
- Add list for About this app screen

### CODE REVIEW

**Comment by tomoya0x00** [comment]
👀

File: uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/about/AboutThisApp.kt [hunk 1]
```diff
@@ -24,6 +34,12 @@ fun AboutThisApp() {
         AboutThisAppComponent()
         Spacer(modifier = Modifier.height(68.dp))
         WhatIsDroidKaigiComponent()
+        Spacer(modifier = Modifier.height(34.dp))
+        AboutThisAppMenuListComponent(
+            onClickPrivacyPolicy = {},
+            onClickLicense = {},
+            appVersion = "1.0.0" // 動的に取得するよう修正必要
```
Comment (tomoya0x00) [inline uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/about/AboutThisApp.kt:41]: Could you write this comment in English and create an issue about it? 🙏

**Review by tomoya0x00 — COMMENTED** [review]
@akihiro-n Almost LGTM! Could you check my comment 🙇

**Review by takahirom — APPROVED** [review]
LGTM 👍

**Pull request merged**